packmanager/manager.go

```diff
@@ -42,6 +42,8 @@ type PackageManager interface {
 	// Add a source to the package manager
 	AddSource(context.Context, string) error
 
+	Prune(context.Context, ...QueryOption) error
```

Comment:
Can you add a comment preamble for the interface method?

Reply:
Done 👍🏽